### PR TITLE
Adding ODA-API: accessing astronomical data analysis workflows at MMODA

### PR DESCRIPTION
**Pull request by volodymyrss**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by volodymyrss** [comment]
Ready for review, I think. 

@conda-forge-admin, please ping conda-forge/help-python

Thanks!

File: recipes/oda-api/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/oda_api-{{ version }}.tar.gz
+  sha256: 87982756ecfe3ed4ba663cc1685b0b788a545223ada85c6e8287f7b12fcd5b7a
+
+build:
+  entry_points:
```
Comment (ocefpaf) [inline recipes/oda-api/meta.yaml:13]: Can you add a comment here that this package has scripts and b/c of that cannot be noarch? That is so we can revisit it in the future and, if the scripts are gone, make it noarch.

PS: The scrips are declared here in the package setup.py

https://github.com/oda-hub/oda_api/blob/7890a709ffe6de620011583ed2a18d4bf9b883f5/setup.py#L25

Comment (volodymyrss) [reply recipes/oda-api/meta.yaml:13]: We'll try to remove them I think they are obsolete here https://github.com/oda-hub/oda_api/pull/213

File: recipes/oda-api/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{% set name = "oda-api" %}
+{% set version = "1.1.43" %}
```
Comment (volodymyrss) [inline recipes/oda-api/meta.yaml:2]:
```suggestion
{% set version = "1.1.44" %}
```

File: recipes/oda-api/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/oda_api-{{ version }}.tar.gz
+  sha256: 87982756ecfe3ed4ba663cc1685b0b788a545223ada85c6e8287f7b12fcd5b7a
```
Comment (volodymyrss) [inline recipes/oda-api/meta.yaml:10]:
```suggestion
  sha256: af919d2c072452e82e3855b3533d2e6b26fe43d73b5d6108d3ec6998a164b801
```

File: recipes/oda-api/meta.yaml (hunk 1 above)
Comment (volodymyrss) [reply recipes/oda-api/meta.yaml:13]: It is now [removed](https://github.com/oda-hub/oda_api/blob/941a6857c8014882abd9d2a1fa2fab82d4b06f7c/setup.py#L25), only entrypoints remain. Is this sufficient?

Thanks!

Comment (ocefpaf) [reply recipes/oda-api/meta.yaml:13]: Cool. In that case you can make it noarch.

Comment (volodymyrss) [reply recipes/oda-api/meta.yaml:14]: It is done!

**Comment by volodymyrss** [comment]
Let me know what you think @ocefpaf .

**Comment by volodymyrss** [comment]
Thanks a lot @ocefpaf !

**Pull request merged**